.github/workflows/schematic-api-ci.yml

```diff
@@ -119,10 +119,22 @@ jobs:
               yaml.dump(secrets, file)
         shell: python
 
+      - name: Set up google credentials
```

Comment:
Two reasons:
 I couldn't get it to work as an environmental variable. :)
This is also more consistent so that local development, and this github workflow work the same by taking in google credentials as a file.

Reply:
That's fair. I can't see anything wrong by using schematic_service_account_creds.json file. I was just curious about your reasoning of using it.